code/src/components/product-card/ProductCard.tsx

```diff
@@ -41,10 +64,17 @@ const ProductCard = ({ product, onAddToCart }: ProductCardProps) => {
         <AppTypography className="spa-product-card__footer-price">
           {formatPrice(product.price)}
         </AppTypography>
-        <AppButton size="small" variant="shadow" onClick={handleAddToCart}>
-          <AddIcon />
-          <AppTypography translationKey="productCard.add" />
-        </AppButton>
+        <AppIconButton
+          onClick={handleCartIconClick}
+          className={cn(
+            "spa-product-card__cart-button",
+            isProductInCart && "spa-product-card__cart-button_active"
```

Comment:
I suggest using modifier here, like `spa-product-card__cart-button--active`

Reply:
Thank you. I guess there are two approaches for adding modifiers because `_` was used in documentation I read. At the same time, in other tutorials and documentation `--` was used. I changed it as you proposed to keep our codebase consistent.